### ISSUE TITLE
B.3.2.1 name is missing a closing parenthesis

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wai-atag-report-tool#203]
"Repair Assistance (WCAG" is missing a closing parenthesis character.
